File: C#/Урок 039/Задача 01.cs
class Program
{
    static void Main()
    {
        int peopleWhoLiveInCity;

        do
        {
            System.Console.WriteLine("Enter positive number of people who live in your city: ");
            peopleWhoLiveInCity = System.Convert.ToInt32(System.Console.ReadLine());

        } while (peopleWhoLiveInCity <= 0);

        int peopleWhoSpeakUkrainian;

        do
        {
            System.Console.WriteLine("Enter positive number of people who speak ukrainian in your city: ");
            peopleWhoSpeakUkrainian = System.Convert.ToInt32(System.Console.ReadLine());

            if (peopleWhoSpeakUkrainian > peopleWhoLiveInCity)
                System.Console.WriteLine("Number of people speaking ukrainian can't be greater than people living in your city.");

        } while (peopleWhoSpeakUkrainian <= 0 || peopleWhoSpeakUkrainian > peopleWhoLiveInCity);

        int chanceUserSpeaksUkrainian = (int)((double)peopleWhoSpeakUkrainian / peopleWhoLiveInCity * 100); // in %
        System.Console.WriteLine($"There is {chanceUserSpeaksUkrainian}% chance you speak ukrainian.");
    }
}

File: C++/Урок 027 - Логічні оператори/Задача 01.cpp
#include <iostream>

int main()
{
	std::cout << "Enter your number: ";
	int number;
	std::cin >> number;

	const int LOWER_BOUND = -100;
	const int UPPER_BOUND = 100;

	std::cout << "If you see 1 - your number is in the range ";
	std::cout << "[" << LOWER_BOUND << " ... " << UPPER_BOUND << "] \\n";
	std::cout << "If you see 0 - your number is not in the range ";
	std::cout << "[" << LOWER_BOUND << " ... " << UPPER_BOUND << "] \\n";

	std::cout << (number >= LOWER_BOUND && number <= UPPER_BOUND) << std::endl;

	return 0;
}